dashboard/config/scripts/levels/DigitalCitizenship.level

```diff
@@ -0,0 +1,15 @@
+<Unplugged>
```

Comment:
Is this the only level we're seeding from `unplugged.yml`?

Reply:
Good question Bethany! There are a few more we are seeding through unplugged.en.yml but which are unused, such as "Unplug1" through "Unplug11" and "deeper learning description". I confirmed that these are not in any scripts. 

There are also some entries which I removed earlier here: https://github.com/code-dot-org/code-dot-org/commit/915ba98b00113e8e485ca2501180f855566af1c7 I removed those initially because some of those were the ones causing duplicate levels, and I wanted to stop them from being seeded right away. However, I could also have just ignored them, in which case they would have become harmless like Unplug1, etc after this PR went in.

Reply:
to answer a different way, DigitalCitizenship is the only level we're currently seeding from unplugged.en.yml which we are relying on in a live script.

Reply:
Great, thanks for confirming! Sounds like this change is definitely the right approach as well!